src/types.d.ts

```diff
@@ -53,5 +58,7 @@ declare global {
 		recordTrackEvent: ( name: string, options?: any ) => void;
 	};
 
+	var gravatarEnhancedComments : GravatarEnhancedComments;
```

Comment:
Nit: Code formatting:

```suggestion
	var gravatarEnhancedComments: GravatarEnhancedComments;
```